loginpage/js/loginpage.js

```diff
@@ -1,21 +1,21 @@
 $(document).ready(function()
 {
```

Comment:
Opening brace should be on the same line as the declaration

*PHPCodeSnifferBear, severity NORMAL, section `js`.*